src/main/java/com/burda/hw15/task1/MyList.java

```diff
@@ -0,0 +1,39 @@
+package com.burda.hw15.task1;
+
+public class MyList<E> {
+    private MyNode<E> head;
+    private MyNode<E> tail;
+    private int size;
+
+    public void add(E value) {
+        final MyNode<E> newNode = new MyNode<E>(value);
+        if (head == null) {
+            head = newNode;
+        }
+        if (tail != null) {
+            MyNode<E> temp = this.tail;
+            temp.next = newNode;
+            newNode.prev = temp;
+        }
```

Comment:
The same
`if (tail != null) {
            this.tail.next = newNode;
            newNode.prev = this.tail;
        }`